notebooks/shield-dash-import.py

```diff
@@ -22,8 +24,12 @@ def getShieldProps(p):
     }
     for k in ['firstrun', 'msg', 'name', 'variation']:
         if k == 'firstrun':
-            out[k] = datetime.datetime.utcfromtimestamp(
-                int(p['payload'][k]) // 1e3)
+            firstrun = p['payload'].get(k)
+            if firstrun is None:
+                out[k] = out['creation_date']
```

Comment:
I was getting KeyError on 'firstrun' for some of the data, it seems. So I hacked this temporarily. Not sure what the right solution is.